Dockerfile.backend

```diff
@@ -0,0 +1,12 @@
+FROM tiangolo/uwsgi-nginx-flask
```

Comment:
Ain't not sure about this image - I think, image https://hub.docker.com/_/python/ would be much better than another not safe images